### ISSUE TITLE
Exception while invoking method cloud:checkRegisterStatus

### ISSUE THREAD

**Issue by piotrkochan** [RocketChat/Rocket.Chat#19027]
### Description:

After upgrade to 3.6.3 we regularly receiving cloud:checkRegisterStatus exception.
Our instance is properly linked to the cloud, "Cloud Service Privacy Terms Agreement" are checked.

![](https://i.imgur.com/A5sn6Xu.png)

Sync button does not make difference. 

Push notification works (not always but they are)

### Server Setup Information:

- Version of Rocket.Chat Server: 3.6.3
- Operating System: Ubuntu 16
- Deployment Method:  tar
- Number of Running Instances: 1
- DB Replicaset Oplog: On
- NodeJS Version: 12.16
- MongoDB Version: 4

### Client Setup Information

- Desktop App or Browser Version: not relevant
- Operating System: not relevant

### Relevant logs:

```
Exception while invoking method cloud:checkRegisterStatus
Error: Not authorized [error-not-authorized]
    at MethodInvocation.cloud:checkRegisterStatus (app/cloud/server/methods.js:24:10)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1771:12)
    at packages/ddp-server/livedata_server.js:1689:15
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at packages/ddp-server/livedata_server.js:1687:36
    at new Promise (<anonymous>)
    at Server.applyAsync (packages/ddp-server/livedata_server.js:1686:12)
    at Server.apply (packages/ddp-server/livedata_server.js:1625:26)
    at Server.call (packages/ddp-server/livedata_server.js:1607:17)
    at Object.post (app/api/server/v1/misc.js:262:26)
    at app/api/server/api.js:394:82
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at Object._internalRouteActionHandler [as action] (app/api/server/api.js:394:39)
    at Route.share.Route.Route._callEndpoint (packages/nimble_restivus/lib/route.coffee:150:32)
    at packages/nimble_restivus/lib/route.coffee:59:33
    at packages/simple_json-routes.js:98:9
 => awaited here:
    at Promise.await (/opt/Rocket.Chat/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:60:12)
    at Server.apply (packages/ddp-server/livedata_server.js:1638:22)
    at Server.call (packages/ddp-server/livedata_server.js:1607:17)
    at Object.post (app/api/server/v1/misc.js:262:26)
    at app/api/server/api.js:394:82
    at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1234:12)
    at Object._internalRouteActionHandler [as action] (app/api/server/api.js:394:39)
    at Route.share.Route.Route._callEndpoint (packages/nimble_restivus/lib/route.coffee:150:32)
    at packages/nimble_restivus/lib/route.coffee:59:33
    at packages/simple_json-routes.js:98:9
```

**Comment by caesarcxiv** [comment]
Exception while invoking method cloud:checkRegisterStatus Error: Not authorized [error-not-authorized]
Receiving this error in logs repeatedly as well

**Comment by sheuschkel** [comment]
Same for me, Version 3.6.3

**Comment by Roemer** [comment]
I have the same with 3.7.0

**Comment by sampaiodiego** [comment]
looks like it is throwing this error when it shouldn't, looks like this call is being done by non-admins, that's why the "Not authorized" errors..

**Comment by Nono-m0le** [comment]
I do have the same bug (together with https://github.com/RocketChat/Rocket.Chat/issues/18662 / https://github.com/RocketChat/Rocket.Chat/issues/18782 )

**Comment by cprice-kgi** [comment]
also experiencing this error in the logs with version 3.6.0.

**Comment by theabhinavdas** [comment]
Facing this bug on version 3.9.3 right now. As well as the blue banner that asks us to join the push notification service.

**Comment by mr-karan** [comment]
Why is this closed? It's happening on 3.10.0 as well which is the latest on Docker Hub.

**Comment by IARI** [comment]
I have this problem on 4.4.0
